### ISSUE TITLE
Race Condition/Deadlock? after upgrade to 0.24.0

### ISSUE THREAD

**Issue by aaronjwhiteside** [fabric8io/docker-maven-plugin#981]
### Description
After upgrading to 0.24.0, I find that about 1 out of every 3 builds experiences this hanging issue, where the `<wait><log>` appears to be matched, but the plugin does not proceed any further.

From the provided stack trace you can see that the `main` thread is waiting on a lock that is held by `Thread-20`.. this thread never appears to terminate, I have to ^C the build to recover.

Plugin Configuration
```xml
                        <image>
                            <alias>couchbase</alias>
                            <name>yapstone/dev/couchbase:${couchbase-docker-image.version}</name>
                            <run>
                                <env>
                                    <BUCKET1_NAME>YS_Apps</BUCKET1_NAME>
                                    <BUCKET2_NAME>YS_Secure</BUCKET2_NAME>

                                    <USER1_NAME>UserProfile</USER1_NAME>

                                    <USER2_NAME>UserAuth</USER2_NAME>
                                    <USER2_PASSWORD>password</USER2_PASSWORD>
                                    <USER2_ROLES>data_reader[YS_Secure],data_writer[YS_Secure],fts_searcher[YS_Secure],query_insert[YS_Secure],query_select[YS_Secure],query_delete[YS_Secure],query_update[YS_Secure]</USER2_ROLES>
                                </env>

                                <volumes>
                                    <bind>${project.build.directory}/couchbase-logs:/opt/couchbase/var/lib/couchbase/logs</bind>
                                </volumes>

                                <!-- wait for image to finish initializing. 60 seconds should be enough -->
                                <wait>
                                    <log>Setup finished</log>
                                    <time>60000</time>
                                </wait>
                            </run>
                        </image>
```

Output from maven run
```
[INFO] DOCKER> [yapstone/dev/couchbase:enterprise-5.1.0-2018-03-01] "couchbase": Start container 500498979a53
couchbase> ++ '[' '!' -f /opt/couchbase/var/lib/couchbase/logs/info.log ']'
couchbase> ++ /entrypoint.sh couchbase-server
couchbase> ++ sleep 0.2
couchbase> Starting Couchbase Server -- Web UI available at http://<ip>:8091
couchbase> and logs available in /opt/couchbase/var/lib/couchbase/logs
couchbase> ++ '[' '!' -f /opt/couchbase/var/lib/couchbase/logs/info.log ']'
couchbase> ++ sleep 0.2
couchbase> ++ '[' '!' -f /opt/couchbase/var/lib/couchbase/logs/info.log ']'
couchbase> ++ sleep 0.2
couchbase> ++ '[' '!' -f /opt/couchbase/var/lib/couchbase/logs/info.log ']'
couchbase> ++ sleep 0.2
couchbase> ++ '[' '!' -f /opt/couchbase/var/lib/couchbase/logs/info.log ']'
couchbase> ++ sleep 0.2
couchbase> ++ '[' '!' -f /opt/couchbase/var/lib/couchbase/logs/info.log ']'
couchbase> ++ sleep 0.2
couchbase> ++ '[' '!' -f /opt/couchbase/var/lib/couchbase/logs/info.log ']'
couchbase> ++ sleep 0.2
consul>     2018/03/28 04:20:34 [DEBUG] Skipping remote check "serfHealth" since it is managed automatically
consul>     2018/03/28 04:20:34 [DEBUG] agent: Node info in sync
consul>     2018/03/28 04:20:34 [DEBUG] agent: Node info in sync
couchbase> ++ '[' '!' -f /opt/couchbase/var/lib/couchbase/logs/info.log ']'
couchbase> ++ sleep 0.2
couchbase> ++ '[' '!' -f /opt/couchbase/var/lib/couchbase/logs/info.log ']'
couchbase> ++ sleep 0.2
couchbase> ++ '[' '!' -f /opt/couchbase/var/lib/couchbase/logs/info.log ']'
couchbase> ++ sleep 0.2
couchbase> ++ '[' '!' -f /opt/couchbase/var/lib/couchbase/logs/info.log ']'
couchbase> ++ sleep 0.2
couchbase> ++ '[' '!' -f /opt/couchbase/var/lib/couchbase/logs/info.log ']'
couchbase> ++ sleep 0.2
couchbase> ++ '[' '!' -f /opt/couchbase/var/lib/couchbase/logs/info.log ']'
couchbase> ++ sleep 0.2
couchbase> ++ '[' '!' -f /opt/couchbase/var/lib/couchbase/logs/info.log ']'
couchbase> ++ sleep 0.2
couchbase> ++ '[' '!' -f /opt/couchbase/var/lib/couchbase/logs/info.log ']'
couchbase> ++ sleep 0.2
couchbase> ++ '[' '!' -f /opt/couchbase/var/lib/couchbase/logs/info.log ']'
couchbase> ++ sleep 0.2
couchbase> ++ '[' '!' -f /opt/couchbase/var/lib/couchbase/logs/info.log ']'
couchbase> ++ grep 'Couchbase Server has started' /opt/couchbase/var/lib/couchbase/logs/info.log
couchbase> ++ sleep 0.5
couchbase> ++ grep 'Couchbase Server has started' /opt/couchbase/var/lib/couchbase/logs/info.log
couchbase> ++ sleep 0.5
couchbase> ++ grep 'Couchbase Server has started' /opt/couchbase/var/lib/couchbase/logs/info.log
couchbase> ++ sleep 0.5
couchbase> ++ grep 'Couchbase Server has started' /opt/couchbase/var/lib/couchbase/logs/info.log
couchbase> ++ sleep 0.5
couchbase> ++ grep 'Couchbase Server has started' /opt/couchbase/var/lib/couchbase/logs/info.log
couchbase> ++ sleep 0.5
couchbase> ++ grep 'Couchbase Server has started' /opt/couchbase/var/lib/couchbase/logs/info.log
couchbase> ++ sleep 0.5
couchbase> ++ grep 'Couchbase Server has started' /opt/couchbase/var/lib/couchbase/logs/info.log
couchbase> ++ sleep 0.5
couchbase> ++ grep 'Couchbase Server has started' /opt/couchbase/var/lib/couchbase/logs/info.log
couchbase> ++ sleep 0.5
couchbase> ++ grep 'Couchbase Server has started' /opt/couchbase/var/lib/couchbase/logs/info.log
couchbase> ++ sleep 0.5
couchbase> ++ grep 'Couchbase Server has started' /opt/couchbase/var/lib/couchbase/logs/info.log
couchbase> ++ sleep 0.5
couchbase> ++ grep 'Couchbase Server has started' /opt/couchbase/var/lib/couchbase/logs/info.log
couchbase> [user:info,2018-03-28T04:20:41.143Z,ns_1@127.0.0.1:ns_server_sup<0.240.0>:menelaus_sup:start_link:46]Couchbase Server has started on web port 8091 on node 'ns_1@127.0.0.1'. Version: "5.1.0-5552-enterprise".
couchbase> ++ couchbase-cli cluster-init -c 127.0.0.1:8091 --cluster-username=Administrator --cluster-password=password --cluster-ramsize=256 --cluster-index-ramsize=256 --cluster-fts-ramsize=256 --services=data,index,query,fts
couchbase> SUCCESS: Cluster initialized
couchbase> +++ hostname
couchbase> ++ couchbase-cli setting-cluster -c 127.0.0.1:8091 -u Administrator -p password --cluster-name=500498979a53
couchbase> SUCCESS: Cluster settings modified
couchbase> ++ '[' -z '' ']'
couchbase> +++ hostname -I
couchbase> ++ NODE_ADDRESS='172.17.0.3 '
couchbase> ++ couchbase-cli node-init -c 127.0.0.1:8091 -u Administrator -p password --node-init-hostname=172.17.0.3
couchbase> SUCCESS: Node initialized
couchbase> ++ [[ data,index,query,fts =~ .*index.* ]]
couchbase> ++ '[' '!' -f /opt/couchbase/var/lib/couchbase/logs/indexer.log ']'
couchbase> ++ grep 'Indexer::NewIndexer Status Active' /opt/couchbase/var/lib/couchbase/logs/indexer.log
couchbase> ++ sleep 0.2
couchbase> ++ grep 'Indexer::NewIndexer Status Active' /opt/couchbase/var/lib/couchbase/logs/indexer.log
couchbase> ++ sleep 0.2
couchbase> ++ grep 'Indexer::NewIndexer Status Active' /opt/couchbase/var/lib/couchbase/logs/indexer.log
couchbase> 2018-03-28T04:20:43.368+00:00 [Info] Indexer::NewIndexer Status Active
couchbase> ++ [[ data,index,query,fts =~ .*query.* ]]
couchbase> ++ '[' '!' -f /opt/couchbase/var/lib/couchbase/logs/query.log ']'
couchbase> ++ grep 'Listen Address' /opt/couchbase/var/lib/couchbase/logs/query.log
couchbase> _time=2018-03-28T04:20:43.234+00:00 _level=INFO _msg=HttpEndpoint: Listen Address=[::]:8093
couchbase> ++ COUNTER=1
couchbase> ++ BUCKET_NAME=BUCKET1_NAME
couchbase> ++ [[ -n YS_Apps ]]
couchbase> ++ BUCKET_RAM_SIZE=BUCKET1_RAM_SIZE
couchbase> ++ echo 'Creating Bucket: YS_Apps'
couchbase> ++ couchbase-cli bucket-create -c 127.0.0.1:8091 -u Administrator -p password --bucket=YS_Apps --bucket-ramsize=128 --bucket-replica=0 --bucket-type=couchbase --bucket-eviction-policy=fullEviction --wait
couchbase> Creating Bucket: YS_Apps
couchbase> SUCCESS: Bucket created
couchbase> ++ let COUNTER++
couchbase> ++ BUCKET_NAME=BUCKET2_NAME
couchbase> ++ [[ -n YS_Secure ]]
couchbase> ++ BUCKET_RAM_SIZE=BUCKET2_RAM_SIZE
couchbase> Creating Bucket: YS_Secure
couchbase> ++ echo 'Creating Bucket: YS_Secure'
couchbase> ++ couchbase-cli bucket-create -c 127.0.0.1:8091 -u Administrator -p password --bucket=YS_Secure --bucket-ramsize=128 --bucket-replica=0 --bucket-type=couchbase --bucket-eviction-policy=fullEviction --wait
couchbase> SUCCESS: Bucket created
couchbase> ++ let COUNTER++
couchbase> ++ BUCKET_NAME=BUCKET3_NAME
couchbase> ++ [[ -n '' ]]
couchbase> ++ COUNTER=1
couchbase> ++ USER_NAME=USER1_NAME
couchbase> ++ [[ -n UserProfile ]]
couchbase> ++ USER_PASSWORD=USER1_PASSWORD
couchbase> ++ USER_ROLES=USER1_ROLES
couchbase> ++ echo 'Creating User: UserProfile'
couchbase> Creating User: UserProfile
couchbase> ++ couchbase-cli user-manage -c 127.0.0.1:8091 -u Administrator -p password --set --rbac-name=UserProfile --rbac-username=UserProfile --rbac-password=password '--roles=data_reader[YS_Apps],data_writer[YS_Apps],fts_searcher[YS_Apps],query_insert[YS_Apps],query_select[YS_Apps],query_delete[YS_Apps],query_update[YS_Apps]' --auth-domain=local
couchbase> SUCCESS: RBAC user set
couchbase> ++ let COUNTER++
couchbase> ++ USER_NAME=USER2_NAME
couchbase> ++ [[ -n UserAuth ]]
couchbase> ++ USER_PASSWORD=USER2_PASSWORD
couchbase> ++ USER_ROLES=USER2_ROLES
couchbase> ++ echo 'Creating User: UserAuth'
couchbase> Creating User: UserAuth
couchbase> ++ couchbase-cli user-manage -c 127.0.0.1:8091 -u Administrator -p password --set --rbac-name=UserAuth --rbac-username=UserAuth --rbac-password=password '--roles=data_reader[YS_Secure],data_writer[YS_Secure],fts_searcher[YS_Secure],query_insert[YS_Secure],query_select[YS_Secure],query_delete[YS_Secure],query_update[YS_Secure]' --auth-domain=local
couchbase> SUCCESS: RBAC user set
couchbase> ++ let COUNTER++
couchbase> ++ USER_NAME=USER3_NAME
[INFO] DOCKER> Pattern 'Setup finished' matched for container 500498979a53
couchbase> ++ [[ -n '' ]]
couchbase> Setup finished.
couchbase> /entrypoint.sh couchbase-server
couchbase> ++ echo 'Setup finished.'
couchbase> ++ fg 1
```

Stacktrace
```
2018-03-27 21:21:48
Full thread dump Java HotSpot(TM) 64-Bit Server VM (25.141-b15 mixed mode):

"Attach Listener" #47 daemon prio=9 os_prio=31 tid=0x00007fde142c6800 nid=0x580b waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"Thread-20" #46 prio=5 os_prio=31 tid=0x00007fde1802c000 nid=0xa407 runnable [0x000070000c4ce000]
   java.lang.Thread.State: RUNNABLE
	at com.kenai.jffi.Foreign.invokeN3O1(Native Method)
	at com.kenai.jffi.Invoker.invokeN3(Invoker.java:1061)
	at jnr.enxio.channels.Native$LibC$jnr$ffi$1.read(Unknown Source)
	at jnr.enxio.channels.Native.read(Native.java:95)
	at jnr.enxio.channels.NativeSocketChannel.read(NativeSocketChannel.java:68)
	at sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:59)
	- locked <0x000000076e395808> (a java.lang.Object)
	at sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:109)
	at sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:103)
	- locked <0x000000076e395d08> (a sun.nio.ch.ChannelInputStream)
	at java.io.FilterInputStream.read(FilterInputStream.java:133)
	at org.apache.http.impl.io.SessionInputBufferImpl.streamRead(SessionInputBufferImpl.java:137)
	at org.apache.http.impl.io.SessionInputBufferImpl.fillBuffer(SessionInputBufferImpl.java:153)
	at org.apache.http.impl.io.SessionInputBufferImpl.readLine(SessionInputBufferImpl.java:282)
	at org.apache.http.impl.io.ChunkedInputStream.getChunkSize(ChunkedInputStream.java:264)
	at org.apache.http.impl.io.ChunkedInputStream.nextChunk(ChunkedInputStream.java:225)
	at org.apache.http.impl.io.ChunkedInputStream.read(ChunkedInputStream.java:184)
	at org.apache.http.impl.io.ChunkedInputStream.read(ChunkedInputStream.java:213)
	at org.apache.http.impl.io.ChunkedInputStream.close(ChunkedInputStream.java:315)
	at org.apache.http.impl.execchain.ResponseEntityProxy.streamClosed(ResponseEntityProxy.java:143)
	at org.apache.http.conn.EofSensorInputStream.checkClose(EofSensorInputStream.java:228)
	at org.apache.http.conn.EofSensorInputStream.close(EofSensorInputStream.java:172)
	at io.fabric8.maven.docker.access.log.LogRequestor.parseResponse(LogRequestor.java:199)
	at io.fabric8.maven.docker.access.log.LogRequestor.run(LogRequestor.java:108)
	- locked <0x000000076cae4180> (a java.lang.Object)

"Thread-19" #45 prio=5 os_prio=31 tid=0x00007fde1868e000 nid=0xa333 runnable [0x000070000c3cb000]
   java.lang.Thread.State: RUNNABLE
	at com.kenai.jffi.Foreign.invokeN3O1(Native Method)
	at com.kenai.jffi.Invoker.invokeN3(Invoker.java:1061)
	at jnr.enxio.channels.Native$LibC$jnr$ffi$1.read(Unknown Source)
	at jnr.enxio.channels.Native.read(Native.java:95)
	at jnr.enxio.channels.NativeSocketChannel.read(NativeSocketChannel.java:68)
	at sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:59)
	- locked <0x000000076dcec888> (a java.lang.Object)
	at sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:109)
	at sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:103)
	- locked <0x000000076dcecd88> (a sun.nio.ch.ChannelInputStream)
	at java.io.FilterInputStream.read(FilterInputStream.java:133)
	at org.apache.http.impl.io.SessionInputBufferImpl.streamRead(SessionInputBufferImpl.java:137)
	at org.apache.http.impl.io.SessionInputBufferImpl.fillBuffer(SessionInputBufferImpl.java:153)
	at org.apache.http.impl.io.SessionInputBufferImpl.readLine(SessionInputBufferImpl.java:282)
	at org.apache.http.impl.io.ChunkedInputStream.getChunkSize(ChunkedInputStream.java:264)
	at org.apache.http.impl.io.ChunkedInputStream.nextChunk(ChunkedInputStream.java:225)
	at org.apache.http.impl.io.ChunkedInputStream.read(ChunkedInputStream.java:184)
	at org.apache.http.conn.EofSensorInputStream.read(EofSensorInputStream.java:135)
	at com.google.common.io.ByteStreams.read(ByteStreams.java:859)
	at io.fabric8.maven.docker.access.log.LogRequestor.readFully(LogRequestor.java:133)
	at io.fabric8.maven.docker.access.log.LogRequestor.readStreamFrame(LogRequestor.java:149)
	at io.fabric8.maven.docker.access.log.LogRequestor.parseResponse(LogRequestor.java:193)
	at io.fabric8.maven.docker.access.log.LogRequestor.run(LogRequestor.java:108)
	- locked <0x000000076cadbd78> (a java.lang.Object)

"Thread-17" #43 prio=5 os_prio=31 tid=0x00007fde1380a000 nid=0xa533 runnable [0x000070000c2c8000]
   java.lang.Thread.State: RUNNABLE
	at com.kenai.jffi.Foreign.invokeN3O1(Native Method)
	at com.kenai.jffi.Invoker.invokeN3(Invoker.java:1061)
	at jnr.enxio.channels.Native$LibC$jnr$ffi$1.read(Unknown Source)
	at jnr.enxio.channels.Native.read(Native.java:95)
	at jnr.enxio.channels.NativeSocketChannel.read(NativeSocketChannel.java:68)
	at sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:59)
	- locked <0x000000076cfb2ed0> (a java.lang.Object)
	at sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:109)
	at sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:103)
	- locked <0x000000076cfb33d0> (a sun.nio.ch.ChannelInputStream)
	at java.io.FilterInputStream.read(FilterInputStream.java:133)
	at org.apache.http.impl.io.SessionInputBufferImpl.streamRead(SessionInputBufferImpl.java:137)
	at org.apache.http.impl.io.SessionInputBufferImpl.fillBuffer(SessionInputBufferImpl.java:153)
	at org.apache.http.impl.io.SessionInputBufferImpl.readLine(SessionInputBufferImpl.java:282)
	at org.apache.http.impl.io.ChunkedInputStream.getChunkSize(ChunkedInputStream.java:264)
	at org.apache.http.impl.io.ChunkedInputStream.nextChunk(ChunkedInputStream.java:225)
	at org.apache.http.impl.io.ChunkedInputStream.read(ChunkedInputStream.java:184)
	at org.apache.http.conn.EofSensorInputStream.read(EofSensorInputStream.java:135)
	at com.google.common.io.ByteStreams.read(ByteStreams.java:859)
	at io.fabric8.maven.docker.access.log.LogRequestor.readFully(LogRequestor.java:133)
	at io.fabric8.maven.docker.access.log.LogRequestor.readStreamFrame(LogRequestor.java:149)
	at io.fabric8.maven.docker.access.log.LogRequestor.parseResponse(LogRequestor.java:193)
	at io.fabric8.maven.docker.access.log.LogRequestor.run(LogRequestor.java:108)
	- locked <0x000000076ca14410> (a java.lang.Object)

"Service Thread" #9 daemon prio=9 os_prio=31 tid=0x00007fde12002800 nid=0x4003 runnable [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"C1 CompilerThread3" #8 daemon prio=9 os_prio=31 tid=0x00007fde12015000 nid=0x3d03 waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"C2 CompilerThread2" #7 daemon prio=9 os_prio=31 tid=0x00007fde12014800 nid=0x3c03 waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"C2 CompilerThread1" #6 daemon prio=9 os_prio=31 tid=0x00007fde1185a800 nid=0x4303 waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"C2 CompilerThread0" #5 daemon prio=9 os_prio=31 tid=0x00007fde13040800 nid=0x4503 waiting on condition [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"Signal Dispatcher" #4 daemon prio=9 os_prio=31 tid=0x00007fde1301e000 nid=0x3903 runnable [0x0000000000000000]
   java.lang.Thread.State: RUNNABLE

"Finalizer" #3 daemon prio=8 os_prio=31 tid=0x00007fde13807000 nid=0x4d03 in Object.wait() [0x000070000bab1000]
   java.lang.Thread.State: WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	- waiting on <0x00000006c0d831a8> (a java.lang.ref.ReferenceQueue$Lock)
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:143)
	- locked <0x00000006c0d831a8> (a java.lang.ref.ReferenceQueue$Lock)
	at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:164)
	at java.lang.ref.Finalizer$FinalizerThread.run(Finalizer.java:209)

"Reference Handler" #2 daemon prio=10 os_prio=31 tid=0x00007fde13804000 nid=0x4e03 in Object.wait() [0x000070000b9ae000]
   java.lang.Thread.State: WAITING (on object monitor)
	at java.lang.Object.wait(Native Method)
	- waiting on <0x00000006c1109000> (a java.lang.ref.Reference$Lock)
	at java.lang.Object.wait(Object.java:502)
	at java.lang.ref.Reference.tryHandlePending(Reference.java:191)
	- locked <0x00000006c1109000> (a java.lang.ref.Reference$Lock)
	at java.lang.ref.Reference$ReferenceHandler.run(Reference.java:153)

"main" #1 prio=5 os_prio=31 tid=0x00007fde12003800 nid=0x2303 waiting for monitor entry [0x000070000af8e000]
   java.lang.Thread.State: BLOCKED (on object monitor)
	at io.fabric8.maven.docker.access.log.LogRequestor.finish(LogRequestor.java:223)
	- waiting to lock <0x000000076cae4180> (a java.lang.Object)
	at io.fabric8.maven.docker.wait.LogWaitChecker.cleanUp(LogWaitChecker.java:45)
	at io.fabric8.maven.docker.wait.WaitUtil.cleanup(WaitUtil.java:80)
	at io.fabric8.maven.docker.wait.WaitUtil.wait(WaitUtil.java:64)
	at io.fabric8.maven.docker.service.WaitService.wait(WaitService.java:53)
	at io.fabric8.maven.docker.StartMojo$1.call(StartMojo.java:258)
	at io.fabric8.maven.docker.StartMojo$1.call(StartMojo.java:244)
	at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:117)
	at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:38)
	at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:77)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at com.google.common.util.concurrent.MoreExecutors$DirectExecutorService.execute(MoreExecutors.java:260)
	at java.util.concurrent.ExecutorCompletionService.submit(ExecutorCompletionService.java:181)
	at io.fabric8.maven.docker.StartMojo.startImage(StartMojo.java:244)
	at io.fabric8.maven.docker.StartMojo.executeInternal(StartMojo.java:140)
	- locked <0x00000006c29128f8> (a io.fabric8.maven.docker.StartMojo)
	at io.fabric8.maven.docker.AbstractDockerMojo.execute(AbstractDockerMojo.java:221)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:154)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:146)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:305)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:956)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:290)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:194)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)

"VM Thread" os_prio=31 tid=0x00007fde11859000 nid=0x4f03 runnable 

"GC task thread#0 (ParallelGC)" os_prio=31 tid=0x00007fde13009000 nid=0x1c07 runnable 

"GC task thread#1 (ParallelGC)" os_prio=31 tid=0x00007fde13009800 nid=0x1e03 runnable 

"GC task thread#2 (ParallelGC)" os_prio=31 tid=0x00007fde11800800 nid=0x2a03 runnable 

"GC task thread#3 (ParallelGC)" os_prio=31 tid=0x00007fde11802800 nid=0x2b03 runnable 

"GC task thread#4 (ParallelGC)" os_prio=31 tid=0x00007fde1300a000 nid=0x5303 runnable 

"GC task thread#5 (ParallelGC)" os_prio=31 tid=0x00007fde1300b000 nid=0x2d03 runnable 

"GC task thread#6 (ParallelGC)" os_prio=31 tid=0x00007fde11803800 nid=0x2e03 runnable 

"GC task thread#7 (ParallelGC)" os_prio=31 tid=0x00007fde11808800 nid=0x3003 runnable 

"VM Periodic Task Thread" os_prio=31 tid=0x00007fde12003000 nid=0x3f03 waiting on condition 

JNI global references: 345
```

### Info
* d-m-p version : 0.24.0
* Maven version (`mvn -v`) :
```
# mvn --version
Apache Maven 3.5.3 (3383c37e1f9e9b3bc3df5050c29c8aff9f295297; 2018-02-24T11:49:05-08:00)
Maven home: /opt/apache-maven-3.5.3
Java version: 1.8.0_141, vendor: Oracle Corporation
Java home: /Library/Java/JavaVirtualMachines/jdk1.8.0_141.jdk/Contents/Home/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "10.13.3", arch: "x86_64", family: "mac"
```
* Docker version : 18.03.0-ce

**Comment by rhuss** [comment]
thanks for the report and information. It's a duplicate of #767 and I hope to fix this soonish (e.g. with #965). Let's continue over there.
